src/integration-tests/introspector/introspectTest.sh

```diff
@@ -393,6 +420,48 @@ function deployCreateDomainJobPod() {
   waitForPod $pod_name
 }
 
+#
+# Create the model in image docker image
+#
+function createMII_Image() {
+  trace "Info: Create MII Image"
+
+  mkdir -p ${SCRIPTPATH}/mii/workdir/models || exit 1
+  cp ${SCRIPTPATH}/mii/models/*  ${SCRIPTPATH}/mii/workdir/models || exit 1
+  cd ${SCRIPTPATH}/mii/workdir || exit 1
+  export WORKDIR=${SCRIPTPATH}/mii/workdir || exit 1
+  export MODEL_IMAGE_TAG=it || exit 1
+  export MODEL_IMAGE_NAME=model-in-image || exit 1
+
+  docker rmi ${MODEL_IMAGE_NAME}:${MODEL_IMAGE_TAG} --force > /dev/null 2>&1
+
+  trace "Info: Downloading WDT and WIT..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_download.sh \
+   > ${test_home}/miibuild_download.out 2>&1 || exit 1
+
+  trace "Info: Launching WIT to build the image..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_image_model.sh \
```

Comment:
Right now, by default, this always builds even if a local image with the given image-name/tag already exists - it should probably set MODEL_IMAGE_BUILD to 'when-missing'.  It's my understanding this is the general directive for all tests.

Separately, I've been asked to change the default of this script to 'when-missing' - but I won't get to that until at least next week.

Reply:
no problem, added now.